interp/value.go

```diff
@@ -157,7 +157,12 @@ func genValueAsFunctionWrapper(n *node) func(*frame) reflect.Value {
 		if v.IsNil() {
 			return reflect.New(typ).Elem()
 		}
-		return genFunctionWrapper(v.Interface().(*node))(f)
+		vn := v.Interface().(*node)
+		if vn.rval.IsValid() && vn.rval.Type().Kind() == reflect.Func {
```

Comment:
Is it useless to instead do:
```suggestion
		vn, ok := v.Interface().(*node)
		if ok && vn.rval.IsValid() && vn.rval.Type().Kind() == reflect.Func {
```
?